src/services/executors.ts

```diff
@@ -46,6 +46,16 @@ export const executorsApi = createApi({
         method: 'DELETE',
       }),
     }),
+    updateExecutorDefinition: builder.mutation<any, YamlEditBody>({
+      query: body => {
+        return {
+          url: `/executors/${body.name}`,
+          method: 'PATCH',
+          headers: {'Content-Type': 'text/yaml'},
+          body: body.value,
+        };
+      },
```

Comment:
```suggestion
      query: body => ({
        url: `/executors/${body.name}`,
        method: 'PATCH',
        headers: {'content-type': 'text/yaml'},
        body: body.value,
      }),
```
and similar for others